src/platform_impl/apple/appkit/monitor.rs

```diff
@@ -133,15 +134,43 @@ impl VideoModeHandle {
 #[derive(Clone)]
 pub struct MonitorHandle(CGDirectDisplayID);
 
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+struct MonitorUuid([u8; 16]);
```

Comment:
You shouldn't need this, you can just use `[u8; 16]` (or if you really want, `type MonitorUuid = [u8; 16]`).